src/vma/vma_extra.h

```diff
@@ -715,23 +714,6 @@ struct __attribute__ ((packed)) vma_api_t {
 	 */
 	uint64_t vma_extra_supported_mask;
 
-	/**
-	 * get dpcp devices allocated by VMA
-	 *
-	 * @param devices - pointer to write the data to. can be NULL see notes
-	 * @param devices_num - IN\OUT parameter
-	 * 	IN - devices size given by user
-	 * 	OUT- devices returned to user
-	 *
-	 * @return 0 on success -1 otherwise
-	 *
-	 * @note application can:
-	 * call twice once with devices == NULL and get the size needed to allocate
-	 * and call again to get the filled device array.
-	 * if application is called with big enough buffer VMA will update the
-	 * size actually used.
-	 */
-	int (*get_dpcp_devices)(uintptr_t **devices, size_t *devices_num);
```

Comment:
This is a not backward-compatible change in user API. Please add a stub for this function that always return -1.

Reply:
Formally yes, but arguable has low risk in this specific case.

Reply:
We don't need to backward support this API and this is the last element in the struct.

Reply:
There are two bad scenarios:
1. Old Rivermax and new dpcp: Rivermax will try to execute code from address in memory out of this structure. Address will contain some garbage so the behavior in unpredictable.
2. Extension: if in a future you decide to add new member or function into this struct, it will be in the same place in memory as removed one. Can also result in unpredictable behaviour.